indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/supervisor/SeekableStreamSupervisor.java

```diff
@@ -1009,6 +1008,22 @@ public Optional<TaskStatus> getTaskStatus(String id)
     this.taskClient = taskClientFactory.build(dataSource, taskInfoProvider, this.tuningConfig, workerExec);
   }
 
+  public static int calculateWorkerThreads(
```

Comment:
Please add a short javadoc about what these worker threads are used for and how their number is computed.